### PR TITLE
New package: please-0.4.1

### PR DESCRIPTION
**Pull request by edneville**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

This was previously raised in #27037 but has since been reviewed by the SUSE security team.

### CODE REVIEW

File: srcpkgs/pleaser/template [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Template file for 'pleaser'
+pkgname=pleaser
```
Comment (abenson) [inline srcpkgs/pleaser/template:2]: Why not just call it `please`?

File: srcpkgs/pleaser/template [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+_binary=please
+version=0.4.1
+revision=1
+wrksrc="${_binary}-v${version}"
+build_style=cargo
+makedepends="cmake llvm clang pam-devel"
```
Comment (abenson) [inline srcpkgs/pleaser/template:8]: > `cmake llvm clang`

These aren't needed

File: srcpkgs/pleaser/template [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+short_desc="Polite and easy to configure regex-first sudo alternative"
+maintainer="Ed Neville <ed@s5h.net>"
+license="GPL-3.0-or-later"
+homepage="https://www.usenix.org.uk/content/please.html"
+distfiles="https://gitlab.com/edneville/${_binary}/-/archive/v${version}/${_binary}-v${version}.tar.gz"
+checksum=0c81449d57a6bbbf1577d6474349f845e9a8c7a074b27fd1a4ef119ccd4f3c64
+
+pre_build() {
+ export CFLAGS_${RUST_BUILD//-/_}="${CFLAGS_host}"
```
Comment (abenson) [inline srcpkgs/pleaser/template:17]: I'm not sure these are needed, either.

File: srcpkgs/pleaser/template [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+ export CC_${RUST_BUILD//-/_}="${BUILD_CC}"
+}
+
+post_install() {
+ local manpage
+ vdoc README.md
+ manpage=$(find ${wrksrc} -name please.ini.5)
```
Comment (abenson) [inline srcpkgs/pleaser/template:24]: I'd prefer it just be `vman man/please.ini.5` instead of using find

File: srcpkgs/pleaser/template [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+distfiles="https://gitlab.com/edneville/${_binary}/-/archive/v${version}/${_binary}-v${version}.tar.gz"
+checksum=0c81449d57a6bbbf1577d6474349f845e9a8c7a074b27fd1a4ef119ccd4f3c64
+
+pre_build() {
+ export CFLAGS_${RUST_BUILD//-/_}="${CFLAGS_host}"
+ export CC_${RUST_BUILD//-/_}="${BUILD_CC}"
+}
+
+post_install() {
+ local manpage
```
Comment (abenson) [inline srcpkgs/pleaser/template:22]: Indent with a tab instead of the space

**Comment by edneville** [comment]
On 2021-06-11 16:39-0700, Andrew Benson wrote:
> +pkgname=pleaser
> 
> Why not just call it `please`?

@abenson, I'm happy for it to be named please here, which matches 
upstream naming.

Thanks for reviewing the template so far, I believe I've made the other 
corrections that you pointed out.

File: srcpkgs/please/template [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# Template file for 'pleaser'
+pkgname=please
+_binary=please
```
Comment (abenson) [inline srcpkgs/please/template:3]: This is redundant now

File: srcpkgs/please/template [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+build_style=cargo
+makedepends="pam-devel"
+short_desc="Polite and easy to configure regex-first sudo alternative "
+maintainer="Ed Neville <ed@s5h.net>"
+license="GPL-3.0-or-later"
+homepage="https://www.usenix.org.uk/content/please.html"
+distfiles="https://gitlab.com/edneville/${_binary}/-/archive/v${version}/${_binary}-v${version}.tar.gz"
+checksum=0c81449d57a6bbbf1577d6474349f845e9a8c7a074b27fd1a4ef119ccd4f3c64
+
+pre_build() {
```
Comment (abenson) [inline srcpkgs/please/template:16]: I meant the whole pre_build() could go away

File: srcpkgs/please/template [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+	export CC_${RUST_BUILD//-/_}="${BUILD_CC}"
+}
+
+post_install() {
+	local manpage
```
Comment (abenson) [inline srcpkgs/please/template:21]: the variable isn't needed anymore, either

**Comment by edneville** [comment]
> +_binary=please

> +pre_build() {
> +	export CC_${RUST_BUILD//-/_}="${BUILD_CC}"
> +}

> +post_install() {
> +	local manpage

@abenson, I've updated the template, thanks for looking again

**Comment by edneville** [comment]
Hello @abenson, are you happy to merge this now?

**Comment by edneville** [comment]
Hello @abenson, are you happy to merge this now?

**Comment by the-maldridge** [comment]
Thank you for your interest in Void Linux.  After significant internal discussion your request for this package has been rejected due to generally duplicated functionality already present in Void's repos, and concerns related to the security critical nature of the software.

**Pull request closed without merging**